index.js

```diff
@@ -9,9 +9,9 @@
 require('dotenv').config();
 const http = require('http');
 const express = require('express');
-const config = require('./config.js');
+const config = require('./config.tsx');
```

Comment:
it said that in order to add typescript atleast one file needed to be tsx which i just chose this file as it would be the least affected

Reply:
That's possibly the practice for creating a new react app. But I don't think it's required to have .tsc to add TS dependencies in package.json. Do you run into any errors if you don't have .tsc file in project? 
When refactoring or coding in TS, we will still need to have the tsconfig.json or webpack to compile ts to js. 

The config.tsx doesn't look right to me. A .tsx compiles into a .jsx file. But this config file shouldn't be a .jsx. 

If we have to have some .tsx files, perhaps make this branch the main feature branch for future ts work. (Don't merge to master yet). 

@ConalCosgrove  may have more knowledge on the React TS on frontend. Any thoughts?

Reply:
Files that are .js should be converted to .ts rather than .tsx.   
.tsx files are the equivalent of .jsx files, eg. they contain React jsx code. So for config, .ts extension should be good.